### PR TITLE
feat: roll driver to 1.3.0-next.1596843106133, handle paths on the client side

### PR DESCRIPTION
**Pull request by dgozman**

### CODE REVIEW

File: tests/async/test_defaultbrowsercontext.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ async def test_context_cookies_should_work(server, launch_persistent):
             "expires": -1,
             "httpOnly": False,
             "secure": False,
-            "sameSite": "None",
+            "sameSite": "Lax" if is_firefox else "None",
```
Comment (pavelfeldman) [inline tests/async/test_defaultbrowsercontext.py:60]: Why did this change?

Comment (dgozman) [reply tests/async/test_defaultbrowsercontext.py:60]: Andrey says that Firefox has switched the defaults.

**Comment by pavelfeldman** [comment]
We might need to update client then to ensure priper 🍪 roundtrip.

File: tests/async/test_defaultbrowsercontext.py (hunk 1 above)
Comment (pavelfeldman) [reply tests/async/test_defaultbrowsercontext.py:60]: "There is currently no timeline to ship this feature to the release channel of Firefox.". Let's make sure to fix it upstream. Filing a bug

**Comment by dgozman** [comment]
Should we postpone this until upstream is fixed?

**Comment by mxschmitt** [comment]
> Should we postpone this until upstream is fixed?

Pavel was fine with it (via Slack).

**Pull request merged**